### PR TITLE
Reverting minimum version for agent

### PR DESCRIPTION
**Pull request by SatishThareja**
Reverting minimum version of agent required to older values.
Also, executing Publish-TestResults cmdlet with -RunTitle switch
depending upon the availability of $testRunTitle parameter.

### CODE REVIEW

**Comment by bryanmacfarlane** [comment]
Since this is going in 91, make sure VSO externals is updated to the package this drops.

File: Tasks/ANT/ant.ps1 [hunk 1]
```diff
@@ -152,7 +152,11 @@ if($publishJUnitResultsFromAntBuild)
     else
     {
         Write-Verbose "Calling Publish-TestResults"
-        Publish-TestResults -TestRunner "JUnit" -TestResultsFiles $matchingTestResultsFiles -Context $distributedTaskContext -RunTitle $testRunTitle
+		if([string]::IsNullOrEmpty($testRunTitle)) {
```
Comment (acesiddhu) [inline Tasks/ANT/ant.ps1]: nit: indent it in PS way

if
{
    // blah
}
else
{
    // blah
}

**Comment by acesiddhu** [comment]
Can you mention the testing done.

File: Tasks/PublishTestResults/PublishTestResults.ps1 [hunk 2]
```diff
@@ -50,7 +50,11 @@ else
     $publishResultsOption = Convert-String $publishRunAttachments Boolean
     $mergeResults = Convert-String $mergeTestResults Boolean
     Write-Verbose "Calling Publish-TestResults"
-    Publish-TestResults -TestRunner $testRunner -TestResultsFiles $matchingTestResultsFiles -MergeResults $mergeResults -Platform $platform -Configuration $configuration -Context $distributedTaskContext -RunTitle $testRunTitle -PublishRunLevelAttachments $publishResultsOption
+	if([string]::IsNullOrEmpty($testRunTitle)) {
+		Publish-TestResults -TestRunner $testRunner -TestResultsFiles $matchingTestResultsFiles -MergeResults $mergeResults -Platform $platform -Configuration $configuration -Context $distributedTaskContext -PublishRunLevelAttachments $publishResultsOption
```
Comment (acesiddhu) [inline Tasks/PublishTestResults/PublishTestResults.ps1]: I think this is also a new param and we need to take care of that as well

**Comment by acesiddhu** [comment]
will wait for all the replies before I sign off

**Comment by SatishThareja** [comment]
Testing done:

M91 Agent
	               Maven	Ant 	      Gradle	Publish Test(Attachment upload present/not present)
Title Present	Pass	Pass	Pass	Pass/Pass
Title missing	Pass	Pass	Pass	Pass/Pass


M90 Agent
                       Maven      Ant       Gradle       Publish Test(present/not present)
Title Present      Fail       Fail          Fail            Fail/Fail
Title missing     Pass     Pass        Pass         Fail/Pass

**Pull request closed without merging**